velox/expression/tests/CastExprTest.cpp

```diff
@@ -465,6 +472,183 @@ TEST_F(CastExprTest, invalidDate) {
       "date", {"2012-Oct-23"}, {0}, true, false, VARCHAR(), DATE());
 }
 
+TEST_F(CastExprTest, primitiveInvalidCornerCases) {
+  setCastIntByTruncate(false);
+  // To integer.
```

Comment:
This test method is very large. Consider splitting into smaller ones. Ditto the next one.